arch/risc-v/src/common/espressif/Kconfig

```diff
@@ -303,6 +303,12 @@ config ESPRESSIF_USBSERIAL
 	select OTHER_UART_SERIALDRIVER
 	select ARCH_HAVE_SERIAL_TERMIOS
 
+config ESPRESSIF_DEDICATED_GPIO
+	bool "Dedicated GPIO"
+	default n
+	---help---
+		Enable dedicated GPIO support for faster response time
```

Comment:
Please explain what are these dedicated GPIO, like grouping functions, etc. The "---help---" should include all information the users will need to enable or disable this feature